setup.py

```
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from glob import glob
+import os
+import sys
+
+from setuptools import setup, Extension
+try:
+    from Cython.Build import cythonize
+except ImportError:
+    cythonize = None
```

Do not ignore this error. Cython must be installed.